gslab_scons/misc.py

```diff
@@ -354,6 +358,7 @@ def finder(rel_parent_dir, pattern, excluded_dirs = []):
         out_paths = subprocess.check_output(command, shell = True).replace('\r\n', '\n')
         out_paths = out_paths.split('\n')
         out_paths = filter(bool, map(str.strip, out_paths)) # Strip paths and keep non-empty
+        out_paths = fnmatch.filter(out_paths,pattern)

```

Comment:
Space after comma.

Reply:
Used the Auto-Pep-8-ifier, hopefully that'll make everyone equally displeased.